src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/Runner.kt

```diff
@@ -138,10 +141,24 @@ abstract class Runner protected constructor(
      * Is invoked before each actor execution from the specified thread.
      * The invocations are inserted into the generated code.
      */
-    fun onActorStart(iThread: Int) {
+    open fun onActorStart(iThread: Int) {
         strategy.onActorStart(iThread)
     }
 
+    /**
+     * This method is invoked before the actor start.
```

Comment:
In case of detectable execution when crash occurred, the execution is continued from the beginning of the actor's body. To calculate actor calls statistics, we have to count this body entering, so there is a special callback for this purpose.